pipelinewise/cli/schemas/tap.json

```diff
@@ -281,6 +281,19 @@
       "minimum": 0,
       "maximum": 2500
     },
+    "split_large_files": {
+      "type": "boolean"
+    },
+    "split_file_chunk_size_mb": {
+      "type": "integer",
+      "minimum": 0,
```

Comment:
What does a 0 mean in?

Reply:
0 is invalid. Fixed by https://github.com/transferwise/pipelinewise/pull/696/commits/f6d9211ddbc50360be06f566980293c25af5f970